packages/pwa/app/pages/account/constant.js

```diff
@@ -52,18 +52,31 @@ export const navLinks = [
 ]
 
 export const CONFIRMATION_DIALOG_DEFAULT_CONFIG = defineMessages({
-    dialogTitle: {defaultMessage: 'Confirm Action'},
-    confirmationMessage: {defaultMessage: 'Are you sure you want to continue ?'},
-    primaryActionLabel: {defaultMessage: 'Yes'},
-    alternateActionLabel: {defaultMessage: 'No'}
+    dialogTitle: {defaultMessage: 'Confirm Action', id: 'global.account.title.confirm_action'},
```

Comment:
```suggestion
    dialogTitle: {defaultMessage: 'Confirm Action', id: 'confirmation_dialog.title.confirm_action'},
```
The constant name sounds to me it belongs to the `confirmation_dialog` component instead of a global/account pages